azext_iot/central/params.py

```diff
@@ -35,6 +35,13 @@
     "scroll = deliver errors as they arrive, json = summarize results as json, csv = summarize results as csv",
 )
 
+api_version = CLIArgumentType(
+    options_list=["--api-version", "--av"],
+    choices=CaseInsensitiveList([version.value for version in ApiVersion]),
+    default="1.0",
```

Comment:
Better to use value from Enum rather than hard coded str

Reply:
fixed